website/updates/Update1.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
let's give the file names more meaningful names to make it easier to find specific ones, e.g.`/2020-12-08-copyable-release.md`

Reply:
updated